rskj-core/src/main/java/co/rsk/peg/federation/FederationSupportImpl.java

```diff
@@ -481,25 +483,18 @@ public int voteFederationChange(Transaction tx, ABICallSpec callSpec, SignatureC
         return (int) result.getResult();
     }
 
-    private boolean unknownFederationChangeFunction(String calledFunction) {
-        return Arrays.stream(FederationChangeFunction.values()).noneMatch(fedChangeFunction -> fedChangeFunction.getKey().equals(calledFunction));
-    }
-
-    private ABICallVoteResult executeVoteFederationChangeFunction(boolean dryRun, ABICallSpec callSpec, BridgeEventLogger eventLogger) throws BridgeIllegalArgumentException {
+    private ABICallVoteResult executeVoteFederationChangeFunction(boolean dryRun, byte[][] callSpecArguments, FederationChangeFunction federationChangeFunction, BridgeEventLogger eventLogger) throws BridgeIllegalArgumentException {
```

Comment:
Not a fan of receiving a `byte[][]` parameter. I think it let's you pass whatever. Wouldn't it be better to keep passing the ABICallSpec obejct